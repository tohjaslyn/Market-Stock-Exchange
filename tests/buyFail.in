buy trader1 ABC 10 3
add trader1 100
buy trader1 ABC 0 0
exit
