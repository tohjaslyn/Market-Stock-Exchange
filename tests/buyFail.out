$ No such trader in the market.
$ Success.
$ No trades could be made, order added to buy book.
$ Have a nice day.
